src/Microsoft.ML.AutoML/Utils/ArrayMath.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.ML.AutoML
+{
+    public class ArrayMath
```

Comment:
>ArrayMath [](http://example.com/codeflow?start=17&length=9)

Are the things in here used a lot? We have some efficient array math stuff already in ML.NET you could use if this needs to be efficient.

Reply:
Nope, they are more used as a helper method for array-like numeric structure and efficiency is not the top-priority here. Beside I just find out that most of these methods are not used anywhere so just did a few clean up as well.